docs/configuration.md

```diff
@@ -12,12 +12,6 @@ Create `.env` file in a root directory or set environment variables with the fol
 - `STATIC_URL` - URL where the static files are located.
   E.g., if you use an S3 bucket, you should set it to the bucket's URL. By default, Saleor assumes you serve static files from the root of your site at "http://localhost:9000/".
 
-- `SALEOR_APPS_PAGE_PATH` - Path appended to `MARKETPLACE_URL` to render Saleor Apps page (deprecated).
-
-- `SALEOR_APPS_JSON_PATH` - Path appended to `MARKETPLACE_URL` to fetch a list of Saleor Apps as JSON (deprecated in favor of `APPS_MARKETPLACE_API_URI`).
-
-- `APP_TEMPLATE_GALLERY_PATH` - Path appended to `MARKETPLACE_URL` to render App Template Gallery page (deprecated).
-
```

Comment:
Removing old envs from md - later we are going to remove them from code (bigger refactor)